src/jit/rationalize.cpp

```diff
@@ -157,19 +157,20 @@ void Compiler::fgFixupArgTabEntryPtr(GenTreePtr parentCall, GenTreePtr oldArg, G
     }
 }
 
-// Rewrite InitBlk involving SIMD vector into stlcl.var of a SIMD type.
+// Rewrite a non-TYP_STRUCT indirection as GT_IND(GT_LEA(obj.op1)), or as a simple
+// lclVar if possible.
 //
```

Comment:
Does this only handle SIMD operations?
If so the comment should say that instead of "non-TYP_STRUCT"